tests/karma/unit/controllers/contacts-content.js

```diff
@@ -221,5 +236,21 @@ describe('ContactsContentCtrl', function() {
       });
     });
 
+    it('includes subjectIds in reports search so JSON reports are found', function(done) {
+      childContactPerson.patient_id = 'cd';
+      childContactPerson.place_id = 'ef';
+      stubSearch(null, [ { _id: 'ab' } ]);
+      runPersonTest(done, doc, null, function(selected) {
+        chai.expect(selected.reports.length).to.equal(1);
+        chai.expect(selected.reports[0]._id).to.equal('ab');
+        chai.expect(search.callCount).to.equal(1);
+        chai.expect(search.args[0][0]).to.equal('reports');
+        chai.expect(search.args[0][1].subjectIds.length).to.equal(3);
```

Comment:
yeah, I don't like the `expect` syntax that much. I don't find it that much more legible, and I can never remember how to write it. (then again maybe the use of both styles is not helping me remember the syntax!!) 
Does it throw better errors on fail? Or is it just prettiness when writing? Or something else?